File: questApplication/src/main/java/ru/nekit/android/qls/quest/mediator/types/direction/DirectionQuestViewHolder.java
package ru.nekit.android.qls.quest.mediator.types.direction;

import android.content.Context;
import android.support.annotation.NonNull;
import android.view.View;
import android.widget.TextView;

import ru.nekit.android.qls.R;
import ru.nekit.android.qls.utils.ViewHolder;

class DirectionQuestViewHolder extends ViewHolder {

    public TextView rightMessage, wrongMessage;
    public View targetView;

    DirectionQuestViewHolder(@NonNull Context context) {
        super(context, R.layout.ql_direction);
        targetView = view.findViewById(R.id.view_target);
        rightMessage = (TextView) view.findViewById(R.id.tv_message_right);
        wrongMessage = (TextView) view.findViewById(R.id.tv_message_wrong);
    }

}

File: questPresentation/src/main/java/ru/nekit/android/qls/quest/view/mediator/types/direction/DirectionQuestViewHolder.kt
package ru.nekit.android.qls.quest.view.mediator.types.direction

import android.content.Context
import android.view.View

import ru.nekit.android.qls.R
import ru.nekit.android.utils.ViewHolder

//ver 1.0
internal class DirectionQuestViewHolder(context: Context) : ViewHolder(context, R.layout.ql_direction) {

    var targetView: View = view.findViewById(R.id.view_target)

}